src/irmin-pack/unix/stats.ml

```diff
@@ -208,3 +267,42 @@ let get_offset_stats () =
     offset_significance =
       pack_store.appended_offsets + pack_store.appended_hashes;
   }
+
+(** Stats update functions for [File_manager_stats]; FIXME why can't we just expose the
```

Comment:
We started doing this because some stats computation required more work than just an incr. I guess we keep doing it for consistency.